runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java

```diff
@@ -247,6 +253,21 @@ else if (dailyReset)
 		}
 	}
 
+	private void checkArrows(boolean dailyReset)
+	{
+		if (client.getVar(Varbits.DIARY_WESTERN_EASY) == 1)
+		{
+			if (client.getVar(Varbits.DAILY_ARROWS_STATE) == 0)
```

Comment:
`if (dailyReset || client.getVar(Varbits.DAILY_ARROWS_STATE) == 0)`

Both checks have the same body, so there's no reason to have them in separate if branches.

Reply:
I simply matched the file format that was already there. Should I fix the other functions as well? Only in places where this is also true?